fix(test): scope-migrate receipt path assertion fails on Windows

## Summary
Windows full-matrix CI fails in `tests/scope-migrate.test.ts` because the receipt path assertion hardcodes POSIX separators.

## Failure
Job: https://github.com/luxus/pi-hindsight/actions/runs/29167850994/job/86584083087

```
AssertionError: expected 'C:\Users\RUNNER~1\AppData\Local\Temp\…' to contain '.pi/hindsight/scope-migrate-receipt.json'
Received: ...\.pi\hindsight\scope-migrate-receipt.json
```

## Fix
Make the assertion path-separator safe via `path.join`.

## Acceptance
- `tests/scope-migrate.test.ts` does not assume `/` separators
- `npm run check` / targeted test pass

--- a/tests/scope-migrate.test.ts
+++ b/tests/scope-migrate.test.ts
@@ -117,7 +117,7 @@ describe("scope migrate dry-run", () => {
       projectBankId: "kai-coding",
     });
     const receipt = await writeScopeMigrateReceipt(cwd, plan);
-    expect(receipt.receiptPath).toContain(".pi/hindsight/scope-migrate-receipt.json");
+    expect(receipt.receiptPath).toBe(join(cwd, ".pi", "hindsight", "scope-migrate-receipt.json"));
     const raw = JSON.parse(readFileSync(receipt.receiptPath, "utf8")) as { rewrite: string };
     expect(raw.rewrite).toBe("none");
   });

